platform/conf/conf.go

```diff
@@ -40,7 +40,7 @@ type Conf struct {
 func New(userdata string) (*Conf, error) {
 	c := &Conf{}
 
-	ignc, err := v2.Parse([]byte(userdata))
+	ignc, _, err := v2.Parse([]byte(userdata))
```

Comment:
Should I loop through each entry and convert it to a string and then log it if there is an error? It wasn't clear to me what a report tells me. https://github.com/coreos/ignition/blob/master/config/validate/report/report.go